src/app/ui/editor/pixels_movement.cpp

```diff
@@ -171,6 +179,17 @@ PixelsMovement::PixelsMovement(
   }
 }
 
+PixelsMovement::~PixelsMovement()
+{
+  // Restore and Unlock the TilemapMode at the time of the
+  // PixelMovement creation.
```

Comment:
Did you mean destruction?

Reply:
I mean "Restore the TilemapMode that was saved during the PixelMovement build".
Maybe "Restore Tilemap mode" is enough to describe the idea.
I'll change the comment.